Documentation/GeneralConventions/GuidesXml.rst

```diff
@@ -175,59 +429,191 @@ project_home
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. todo: describe interlink mapping more detailed
+..  code-block:: xml
+    :caption: EXT:adminpanel/Documentation/guides.xml (excerpt)
 
-A list of globally available Interlink (formerly "Intersphinx") repositories
-can be found in :ref:`Available default inventories <t3renderguides:available-default-inventories>`
+    <?xml version="1.0" encoding="UTF-8"?>
+    <guides xmlns="https://www.phpdoc.org/guides" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="https://www.phpdoc.org/guides ../vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
+        >
+        <extension class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+                   project-home="https://extensions.typo3.org/extension/adminpanel/"
+                   project-contact="https://typo3.slack.com/archives/C025BQLFA"
+                   project-repository="https://github.com/typo3/typo3"
+                   project-issues="https://forge.typo3.org/projects/typo3cms-core/issues"
+        />
+    </guides>
 
-It is possible, though rarely needed to define custom interlink mappings:
 
-For example:
+.. _settings-guides-project-links-reference-example:
+
+Example: Project links for a reference or guide
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 ..  code-block:: xml
-    :caption: Documentation/guides.xml, excerpt
+    :caption: Reference-TSconfig Documentation/guides.xml (excerpt)
 
-    <inventory id="sphinx" url="https://www.sphinx-doc.org/en/master/"/>
+    <?xml version="1.0" encoding="UTF-8"?>
+    <guides xmlns="https://www.phpdoc.org/guides" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="https://www.phpdoc.org/guides ../vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
+         >
+        <extension class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+                   project-home="https://docs.typo3.org/m/typo3/reference-tsconfig/main/en-us/"
+                   project-contact="https://typo3.slack.com/archives/C028JEPJL"
+                   project-repository="https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TSconfig"
+                   project-issues="https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TSconfig/issues"
+        />
+    </guides>
+
+
+..  _settings-guides-preferred_typo3_version:
+
+typo3-core-preferred
+--------------------
+
+..  confval:: typo3-core-preferred
+    :name: guides-extension-typo3-core-preferred
+    :type: string
+    :Default: `stable`
+
+    You can set the preferred TYPO3 Core version. This version is used to determine
+    the desired version for links to other manuals and system extensions.
+
+    ..  code-block:: xml
+        :caption: Documentation/guides.xml
+
+        <extension class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+                   typo3-core-preferred="12.4" />
```

Comment:
```suggestion
                   typo3-core-preferred="stable" />
```

I suggest to use "stable" as people tend to copy/paste those and will come up later with references to outdated manuals (when not adapted manually).